commit a97875cfaf1961e7323dbfe8b4d5c7a9fcc30aba (refs/changes/99/1999/2)
Author: Jordan Hendricks <jordan.hendricks@joyent.com>
Date:   2017-05-24T23:58:00+00:00 (2 years, 5 months ago)
    
    joyent/node-manta#310 Bump version to 4.4.0 for `mmpu` bug fixes
    Reviewed by: Richard Kiene <richard.kiene@joyent.com>
    Approved by: Richard Kiene <richard.kiene@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index a18a1b6..ac710a4 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,11 @@
 
 ## not yet released
 
+## 4.4.0
+
+- joyent/node-manta#308 `mmpu commit` does not parse options
+- joyent/node-manta#309 MPU tests are out of sync with Muskie master branch implementation
+
 ## 4.3.0
 
 - MANTA-2169: Support multipart upload of a single file to Manta
diff --git a/package.json b/package.json
index 6c131bc..d35566f 100644
--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
         "type": "git",
         "url": "git://github.com/joyent/node-manta.git"
     },
-    "version": "4.3.0",
+    "version": "4.4.0",
     "main": "./lib/index.js",
     "dependencies": {
         "assert-plus": "^1.0.0",
